packages/datadog_flutter_plugin/ios/Classes/DatadogLogsPlugin.swift

```diff
@@ -12,24 +12,31 @@ public class DatadogLogsPlugin: NSObject, FlutterPlugin {
     registrar.addMethodCallDelegate(instance, channel: channel)
   }
 
-  private var logger: Logger?
-  public var isInitialized: Bool { return logger != nil }
+  private var loggerRegistry: [String: Logger] = [:]
 
   override private init() {
     super.init()
   }
 
-  func initialize(withLogger logger: Logger) {
-    self.logger = logger
+  func addLogger(logger: Logger, withHandle handle: String) {
+    loggerRegistry[handle] = logger
   }
 
-  func initialize(configuration: DatadogFlutterConfiguration.LoggingConfiguration) {
+  func createLogger(loggerHandle: String, configuration: DatadogLoggingConfiguration) {
     let builder = Logger.builder
       .sendNetworkInfo(configuration.sendNetworkInfo)
```

Comment:
and I guess iOS SDK has no method for `sendLogsToDatadog` option in a builder?

Reply:
It does. Also a good catch :)